### PR TITLE
Homework dz7

### PR DESCRIPTION
**Pull request by Natanefa**
dz7
ДЗ 7. Лекция 5 Java Basics. Methods
ДЗ 8. Lecture 06. Java Basics. Strings

### CODE REVIEW

File: src/main/java/homework_dz7/hw_task1.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
+package homework_dz7;
+
+import java.util.Scanner;
+
+public class hw_task1 {
+
+    public static double celcius2kelvin(double temp){
+        temp = temp + 273.16;
```
Comment (schvets) [inline src/main/java/homework_dz7/hw_task1.java:8]: it is not necessary to create an additional temp variable, you can write it simpler, for example
`return  temp - 273.16;`
instead 
```
        temp = temp - 273.16;
        return temp;
```

File: src/main/java/homework_dz7/hw_task2.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+        int letters = 0;
+        for (Character letter : str.toCharArray()) {
+            letters = letters +1;
+        }
+        System.out.println("Character count = " + letters);
+        System.out.println("");
+
+//        Pattern pattern = new Pattern("\\d+");
+//        String code = "AX6BYU56UX6CV6BNT7NM";
```
Comment (schvets) [inline src/main/java/homework_dz7/hw_task2.java:53]: delete commented code

File: src/main/java/homework_dz7/hw_task2.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
+package homework_dz7;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class hw_task2 {
+    public static void main(String[] args) {
+        String str = "Here you can find activities to practise your reading skills. Reading will help you to improve your understanding of the language and build your vocabulary.";
```
Comment (schvets) [inline src/main/java/homework_dz7/hw_task2.java:8]: please move each task to separate class

File: src/main/java/homeworks_dz6/hw_task13.java [hunk 4]
```diff
@@ -6,6 +6,7 @@
     public static void main(String[] args) {
         int[] array = new int[10];
```
Comment (schvets) [inline src/main/java/homeworks_dz6/hw_task13.java:7]: remove a file from HW commit

File: src/main/java/lections/lection6.java [hunk 5]
```diff
@@ -2,6 +2,5 @@
 
 public class lection6 {
     public static void main(String[] args) {
```
Comment (schvets) [inline src/main/java/lections/lection6.java:4]: remove a file from HW commit

**Review by schvets — CHANGES_REQUESTED** [review]
please provide fixes according comments

**Comment by Natanefa** [comment]
Done. Check, please

**Pull request merged**